src/helpers/log.ts

```diff
@@ -0,0 +1,12 @@
+export function createLog(scope: string) {
```

Comment:
簡単にドキュメントも足しときますか！こんな感じ？
```suggestion
/** ロガーを作る。 */
export function createLog(scope: string) {
```

書いてて思ったのですが`log`なのは何かを参考にされた感じでしょうか 👀 
Pythonだとデフォルトで入っているのがloggerという名前だったので気になった次第です。
まあ細かいですが、「createLog」だとログを作ることになっちゃうので。

Reply:
log関数を作るのでcreateLogにしてました。特にこだわりはないのでcreateLoggerにします